src/init.cpp

```diff
@@ -1256,10 +1256,12 @@ bool AppInitMain(NodeContext& node)
     InitSignatureCache();
     InitScriptExecutionCache();
 
-    LogPrintf("Using %u threads for script verification\n", nScriptCheckThreads);
     if (nScriptCheckThreads) {
+        LogPrintf("Using %u threads concurrently for script verification\n", nScriptCheckThreads);
         for (int i=0; i<nScriptCheckThreads-1; i++)
             threadGroup.create_thread([i]() { return ThreadScriptCheck(i); });
+    } else {
+        LogPrintf("Using the only thread for script verification\n");
```

Comment:
Concept ACK, my only comment would be that `-par=0` and `-par=1` do the same thing, but log different messages.

Reply:
Not exactly:
https://github.com/bitcoin/bitcoin/blob/08e29473126d5cc4df6d2b3f368c6f6f641c0bd8/src/init.cpp#L380

Reply:
I'm not talking about the input argument, but the already-processed value we end up with here.

Reply:
After this code
https://github.com/bitcoin/bitcoin/blob/08e29473126d5cc4df6d2b3f368c6f6f641c0bd8/src/init.cpp#L1064-L1071

is executed in `AppInitParameterInteraction()`, `nScriptCheckThreads==1` is an impossible value.

Reply:
Ok, so it will never log `Using 1 threads concurrently for script verification`?

Reply:
Sure.